src/mongo-utils.ts

```diff
@@ -16,10 +17,39 @@ export class MongoUtils {
 	): Promise<mongodb.Db> {
 		const log = global.log.module('mongo');
 		log.info(`Connecting to ${MongoUtils.hidePasswordFromURI(url)}...`);
-		global.dbClient = await mongodb.MongoClient.connect(url, options);
-		const db = (global.dbClient as mongodb.MongoClient).db();
+
+		const mongoClient = new mongodb.MongoClient(url, options);
+
+		mongoClient.on('serverOpening', () => {
+			log.info(`-- mongo-utils.ts serverOpening --`);
+		});
+		mongoClient.on('open', () => {
+			log.info(`-- mongo-utils.ts open --`);
+		});
+		mongoClient.on('close', () => {
+			log.info(`-- mongo-utils.ts close --`);
+		});
+		mongoClient.on('serverClosed', () => {
+			log.info(`-- mongo-utils.ts serverClosed --`);
+		});
+		mongoClient.on('reconnect', () => {
+			log.info(`-- mongo-utils.ts reconnect --`);
+		});
+		mongoClient.on('timeout', (e) => {
+			log.warn(`-- mongo-utils.ts timeout --`, e);
```

Comment:
Could you format clean log ? that was only an example.

Be carrefull to not add unknown object as context

Reply:
@BenjD90 done!